.github/workflows/integration-test.yml

```diff
@@ -103,6 +102,7 @@ jobs:
           - test: 'clickhouse'
           - test: 'datadog-agent'
           - test: 'datadog-logs'
+          - test: 'docker'
```

Comment:
Do we want to keep this aligned with the actual component name, `docker_logs`?

Reply:
I personally think it doesn't matter. If tomorrow we do `docker_events`, we won't need other external services than docker itself, that's why I shortened it to `docker`. Also, some services also have sinks, sources, logs, metrics and have a single command (like `prometheus`).